cmd/notation/verify.go

```diff
@@ -35,6 +35,19 @@ func verifyCommand(opts *verifyOpts) *cobra.Command {
 	command := &cobra.Command{
 		Use:   "verify [reference]",
 		Short: "Verifies OCI Artifacts",
+		Long: `
+** This command is in preview and under development. **
```

Comment:
I would remove this line, these commands are no longer under preview, intact its first version.

Reply:
Agree